SofaKernel/modules/SofaCore/src/sofa/core/objectmodel/DDGNode.h

```diff
@@ -70,32 +70,32 @@ class SOFA_CORE_API DDGNode
     virtual void update() = 0;
 
     /// Returns true if the DDGNode needs to be updated
-    SOFA_DEPRECATE_ASPECT("Update your code by removing the ExecParam parameter at function call.")
+    SOFA_ATTRIBUTE_DISABLED_EXECPARAMS()
```

Comment:
To factore at that level I think the macro should be 
SOFA_DEPRECATE_ASPECT_DISABLE_EXECPARAM... so that we keep the fact that we are within the aspect part of and we are now disabling execparam and have shared text.